docs/manual/common/guide/devmode/code/build-services-ports.sbt

```diff
@@ -2,8 +2,15 @@
 lagomServicesPortRange in ThisBuild := PortRange(40000, 45000)
 //#port-range
 
-//#service-port
+//#service-port-java
 lazy val usersImpl = (project in file("usersImpl"))
   .enablePlugins(LagomJava)
   .settings(lagomServicePort := 11000)
-//#service-port
+//#service-port-java
+
+
+//#service-port-scala
+lazy val usersImpl = (project in file("usersImpl"))
```

Comment:
Can't we move the specific snippets to separate files in `/java` and `/scala` folders?

Reply:
That's very strange because I run it locally and had not issue. Probably because running the site doesn't compile those files.

Reply:
@ignasi35 about the split, I think it's ok to have them together. Especially because there are small code fragments and some minor differences. It's easier to maintain if they are together.

Reply:
The object wrapper trick didn't work probably because it's a .sbt file, not a .scala. Split in two files as @ignasi35 suggested.